cmake/templates/pkgConfig.cmake.in

```diff
@@ -78,6 +78,15 @@ else()
   set(@PROJECT_NAME@_PREFIX ${@PROJECT_NAME@_INSTALL_PREFIX})
 endif()
 
+if(CMAKE_CROSSCOMPILING)
+  if("@DEVELSPACE@" STREQUAL "TRUE")
+    # can not disable host filesystem altogether; otherwise files in src/devel are no longer found
+    set(_find_root_path_both CMAKE_FIND_ROOT_PATH_BOTH)
+  else()
+    set(_find_root_path_both)
```

Comment:
Should this be set to an empty value in all other cases? Since the variable is being used later unconditionally.